cmd/integration/integration.go

```diff
@@ -306,37 +306,61 @@ func podRunning(c *client.Client, podNamespace string, podID string) wait.Condit
 }
 
 func runStaticPodTest(c *client.Client, configFilePath string) {
-	manifest := `version: v1beta2
-id: static-pod
+	hostname, _ := os.Hostname()
```

Comment:
@smarterclayton just removed os.Hostname from the integration tests because it was causing issues running them on a mac.